### DISCUSSION TITLE
How to delete a disabled repository？

### DISCUSSION THREAD

**Discussion by Visytudz** [community/community#77042] [Repositories]
### Select Topic Area

Question

### Body

I forked a repository from someone else, and then that source repository was disabled, causing the forked repository to be disabled as well.
Now I want to delete this repository, but I can't see it in the repository list and I have no access to the repository's settings. I can only try to access it with the url and the following page comes up.
So the question is, how can I delete this repository? Any answer is highly appreciated.
![image](https://github.com/community/community/assets/83176441/b1c4aa61-fdc4-4217-9a73-dd9069528dc4)

**Comment by MantasTech** [comment]
Hello,

you posted the solution yourself. Contact Support 🧠

**Comment by bHabawel** [comment]
You can contact support. It helps.

**Comment by DNin01** [comment]
This might work: [use the GitHub CLI to delete the repository](https://cli.github.com/manual/gh_repo_delete).

**Reply by bricle** [reply]
doesn't work now, it promts
HTTP 403: Repository access blocked (https://api.github.com/repos/...)
